docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -5459,7 +5459,7 @@ Version 2.0.42
 ### SQL
 
 * Fixed 'The provided resource group name ... did not match the name in the Url' error when specifying elastic pool name for `sql db copy` and `sql db replica create` commands
-* Allow configuring default sql server by executing `az configure --defaults sql-server=<name>`
```

Comment:
@dksimpson , please take release-notes-azure-cli.md out of this PR.  This file should only be modified by the engineering team.  It is a reflection of a "point in time" i.e. history.

Reply:
Done